client/src/util/testScheduler.ts

```diff
@@ -19,16 +19,31 @@ export interface CourseSection {
 
 /**
  * Given an array of course sections, filter out non Web-Oriented sections (no labs, waiting lists etc)
- * TODO: For testing purposes, I'm only filtering for Web Oriented Course, but there will be other types of valid lectures
  * @param {CourseSection[]} courseSections array of course sections (of all types including lectures, labs, waitlists)
  * @returns {CourseSection[]} An array of course sections which only contains "lecture" type
  * NOTE: My types here are generic because there might be more changes to schema, so don't want to lock in types just yet
  */
 export const filterLectures = (courseSections: CourseSection[]): CourseSection[] => {
-    const lectures = courseSections.filter((coursesection: CourseSection) => coursesection["activity"] === "Web-Oriented Course");
+    const lectures = courseSections.filter(isLectureType);
     return lectures;
 };
 
+/**
+ * helper function for filterLectures
+ * @param {CourseSection} courseSection a course section
+ * @returns {boolean} returns false if a section's type is any of lectureTypes
+ */
+export const isLectureType = (courseSection: CourseSection) => {
+    const activity = courseSection["activity"];
+    const lectureTypes = [
```

Comment:
This confused me a bit, shouldn't this be notLectureTypes? :D

Reply:
oops yeah LOL. forgot to change it from another thing i had. thanks!

Reply:
wait actually could it still be `isLectureType` since filter keeps the elements that pass the filtering function? or is it convention to name the opposite?? words are hard

Reply:
My brain can't handle this atm.... i would say `isLectureType` and `filterActivityTypes` is fine.